Sources/Vitamin/Foundations/Shadows/Shadows.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+import UIKit
+
+public enum VitaminShadow: String {
+    case shadow100
+    case shadow200
+    case shadow300
+    case shadow400
+    case none
+
+    /// blur value of the shadow, and height
+    var blur: CGFloat {
+        switch self {
+        case .shadow100:
+            return 6
+        case .shadow200:
+            return 12
+        case .shadow300:
+            return 24
+        case .shadow400:
+            return 48
+        case .none:
+            return 0
+        }
+    }
+
+    /// size of the shadow, i.e. offset of the light source creating the shadow
+    var size: CGSize {
+        // In fact, in Vitamin, the blur always equals to the height, we reuse the property to avoid dupliication
+        // If this changes in the fiture, we will just have to provide the independant value
+        CGSize(width: 0, height: self.blur)
+    }
+
+    /// Spread value of the shadow
+    /// Always 0 at the lmoment, but could evolve, that is why this property exists
+    var spread: CGFloat {
+        0
+    }
+
+    /// opacity of the shadow
+    var opacity: Float {
+        if self == .none {
+            return 0
+        }
+        return 1
+    }
+
+    /// color of the shadow
+    /// It is not a VitamoinColor, and s the same for dark and light mode
+    var color: UIColor {
+        if self == .none {
+            return UIColor.clear
+        }
+        return UIColor(red: 0, green: 0.325, blue: 0.49, alpha: 0.1)
```

Comment:
Where does this color come from? It's not available in base colors?

Reply:
No, in Figma, it is not a base color, it's a custom color defined with RGBA.
But honestly, I did not took the tile to compare it with all the existing colors to see if it matches one of them.
I'll do that, just to check.

Reply:
After check, it seems to correspond to blue600, but with an alpha of 0.1 instead of 1.
I changed that in the code.

Reply:
It's maybe a Figma issue 😇 @lauthieb 
I think that I could be nice to have this kind of colors in Base colors.